YACS-RCOS/yacs.n
# Save selected courses

Functionality to save the courses the user has currently selected. 

**Testing Procedure:**

1. Open a terminal and curl a user account:
- `curl -d '{"name":"John Doe", "email":"yacs@rpi.edu", "phone":"1234567890", "major":"CS", "password":"123456", "degree":"Undergraduate"}' -X POST _url_/api/user -H "Content-Type: application/json" `
2. Login to the page with your email and password
3. navigate to _url_/admin
4. Upload the Fall, Spring, and Summer 2020 CSV files.
5. Navigate back to the home page and add courses for each semester. 
- Semester can be changed in the footer
- Make sure to select courses for all there subsemesters of the Summer semester
6. Ensure that when you return to a semester, or refresh the page, all of your previously selected courses still appear in both selected courses, and on your schedule.
7. Log out, refresh the page (all selected courses should be gone), then log back in.
8. Ensure all of your selections are still present after this proccess.
9. Celebrate because this testing process is long
